tests/NHapi.NUnit/NHapi.NUnit.csproj

```diff
@@ -62,6 +62,30 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="TestData\BadInputs\aoor1">

```

Comment:
This was previously autogenerated by Visual Studio. I added your suggestion without the .txt, because these are Files with many weird bytes directly copied from the minimized fuzzing results